app/app.js

```diff
@@ -240,7 +245,7 @@ app.post("/api/activity", async (req, res) => {
     res.status(201).send(result.rows[0]);
   } catch (err) {
     if (err.code === "23503") {
```

Comment:
actually can you put this number in a constant so its more clear what it does. It's postgresql foreign key violation